jujugui/static/gui/src/app/utils/bundle-importer.js

```diff
@@ -165,31 +165,10 @@ YUI.add('bundle-importer', function(Y) {
       @param {Object} e The load event from the file load.
     */
     _fileReaderOnload: function(file, e) {
-      var data;
       var notifications = this.db.notifications;
-      // We support dropping a bundle YAML file and a changeSet JSON file onto
-      // the canvas. The JSON file support should only ever be used
-      // for when there is no guiserver is available like in sandbox mode.
       var extension = file.name.split('.').pop();
       var result = e.target.result;
-      if (extension === 'json') {
```

Comment:
I was going by this comment: "The JSON file support should only ever be used for when there is no guiserver is available like in sandbox mode."

I can leave it in if we use it at other times...